cloud-service-providers/azure/azureml/cli/config.sh

```diff
@@ -22,21 +22,23 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 # AzureML Workspace and corresponding container registry related information
-subscription_id="<your-azure-subscription-id>"
-resource_group="<your-resource-group>"
-workspace="<your-azureml-workspace-name>"
-location="<your-azureml-region>" # eg: "southcentralus", "westeurope" etc.
+subscription_id="ab221ca4-f098-422d-ab2f-5073b3851e68"
```

Comment:
can you remove all personal information and revert it back to original?

Reply:
i did not intend to check in the changes to config.sh. I reverted the changes